src/types/python/excelsheet/mod.rs

```diff
@@ -574,6 +601,39 @@ impl ExcelSheet {
             .and_then(|obj| obj.into_bound_py_any(py))
     }
 
+    pub fn to_arrow_with_errors<'py>(&self, py: Python<'py>) -> PyResult<Bound<'py, PyAny>> {
+        let offset = self.offset();
+        let limit = self.limit();
+
+        let res = record_batch_from_data_and_columns_with_errors(
```

Comment:
```suggestion
        let (rb, errors) = record_batch_from_data_and_columns_with_errors(
```

You can use destructuring here to avoid the tuple field accessing below

Reply:
Updated